src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -336,6 +334,19 @@ public Func<EndpointInfo> OtlpServiceHttpEndPointAccessor
         }
     }
 
+    internal static List<string> GetSupportedCultures()
+    {
+        // our localization list comes from https://github.com/dotnet/arcade/blob/89008f339a79931cc49c739e9dbc1a27c608b379/src/Microsoft.DotNet.XliffTasks/build/Microsoft.DotNet.XliffTasks.props#L22
+        var supportedCultures = CultureInfo.GetCultures(CultureTypes.AllCultures)
+            .Where(culture => LocalizedCultures.Contains(culture.Name) || LocalizedCultures.Contains(culture.TwoLetterISOLanguageName))
```

Comment:
Most of the supported cultures are two-letter, so inverting this might result in less overall work:

```suggestion
            .Where(culture => LocalizedCultures.Contains(culture.TwoLetterISOLanguageName) || LocalizedCultures.Contains(culture.Name))
```